Linux/Car_Client/chronoscomm.h

```diff
@@ -265,6 +267,18 @@ typedef struct {
 #define ISO_VALUE_ID_INIT_SUP_STATUS    0x0200
 #define AUX_VALUE_ID_OBJECT_ID          0xA000
 
+//OSEM
+#define VALUE_ID_OSEM_TRANSMITTER_ID 0x0010
+#define VALUE_ID_OSEM_LATITUDE 0x0020
```

Comment:
Where did these values use to be (e.g ISO_VALUE_ID_LAT here) and why didn't you use those instead of defining your own?

Reply:
I realize now that a lot of these values are already defined in Chronoscom.h from line 189 and forward. These could be used instead to avoid additional code.